pkg/operator/controller/garden/components.go

```diff
@@ -693,3 +698,28 @@ func getAPIServerDomains(domains []string) []string {
 	}
 	return apiServerDomains
 }
+
+func (r *Reconciler) newNginxIngressController() (component.DeployWaiter, error) {
+	return sharedcomponent.NewNginxIngress(
+		r.RuntimeClientSet.Client(),
+		r.GardenNamespace,
+		r.GardenNamespace,
+		r.ImageVector,
+		r.RuntimeVersion,
+		map[string]string{
+			"enable-vts-status":            "false",
+			"server-name-hash-bucket-size": "256",
+			"use-proxy-protocol":           "false",
+			"worker-processes":             "2",
+			"allow-snippet-annotations":    "false",
+		},
```

Comment:
Why do we hard-code the config here when the `Garden` API allows to specify it? I think we need something like https://github.com/gardener/gardener/blob/85f5d9da6df9f35063018f5a8d3981a97f35c738/pkg/gardenlet/controller/seed/seed/nginxingress.go#L109-L133